### DISCUSSION TITLE
Question about this project's descrition

### DISCUSSION THREAD

**Discussion by ovizii** [jmcollin78/versatile_thermostat#418] [Q&A] [answered]
Sorry for the vague title, I couldn't come up with a better title.
I was looking for a replacement of Better Thermostat as its giving me too much trouble.

I've been reading up about this project when I came upon this sentence:

`For this type of thermostat, any heating cycles are controlled by the underlying climate entity and not by the Versatile Thermostat itself.`

Could somebody give me an ELI5 please? What exactly does VT do, and what remains to be done by the physical thermostat?

**Comment by jmcollin78** [comment]
ELI5 ?

**Reply by ovizii** [reply]
Explain like I'm 5 ;-)

**Comment by jmcollin78** [comment, marked as answer]
> For this type of thermostat, any heating cycles are controlled by the underlying climate entity and not by the Versatile Thermostat itself.

In VTherm over an existing climate (the type `over_climate`you are refering to), the heating is done by the existing climate. Basically, VTherm "just" control the underlying climate temp setpoint and mode. But it adds some regulation mecanisms which can override the limitation of your undelying climate.

**Reply by ovizii** [reply]
Thanks, I was confused about the part “any heating cycles are controlled by the underlying climate entity”. This made it sound, like control was still on the side of the physical thermostat.

From your decision it sounds like what I am looking for, an alternative to Better Thermostat. 
It sounds like after installing it and creating VTs which represent the physical thermostats, I'll have no further interaction with the physical ones as all the configuring and controlling will be done by VT.

**Reply by jmcollin78** [reply]
>  an alternative to Better Thermostat

It is exactly an alternative to all thermostat: Better Thermostat, Awesone thermostat, Generic Thermostat. A kind of generalization of all.

**Reply by ovizii** [reply]
Awesome, I will give it a try. I wasn't aware there were so many alternatives around.

**Reply by 2fst4u** [reply]
> > For this type of thermostat, any heating cycles are controlled by the underlying climate entity and not by the Versatile Thermostat itself.
> 
> In VTherm over an existing climate (the type `over_climate`you are refering to), the heating is done by the existing climate. Basically, VTherm "just" control the underlying climate temp setpoint and mode. But it adds some regulation mecanisms which can override the limitation of your undelying climate.

Hi there, just wanting to clarify my understanding on this answer, so when you set a temperature for versatile thermostat, will the set point of the underlying climate entity that it is controlling be adjusted if the temperature strays too far from the target you wanted it to be?

In other words if I set 16 in versatile thermostat and the underlying climate entity is set to 16, but the temperature gets to like 17.5, will the climate entity be adjusted to something like 15?

It sounds like that's what you're saying but I'm just elaborating to save confusion.

Edit: nevermind I needed to turn on the self regulation haha ignore me.
